### DISCUSSION TITLE
Display image in the options

### DISCUSSION THREAD

**Discussion by dhruv-icreative** [TarekRaafat/autoComplete.js#275] [Q&A]
Can I display the image in the suggested result inside `<li>` tag?
I want the result like:
`<img /> option1`
`<img /> option2`

**Comment by folknor** [comment]
You can put whatever content you want for each item;
```js
new autoComplete({
	resultItem: {
		element: (element, data) => {
			element.innerHTML = `<img src="${data.value.img}">${data.value.text}`
		}
	},
})
```

**Reply by folknor** [reply]
Take a look at https://github.com/TarekRaafat/autoComplete.js/issues/95#issuecomment-830598444 and https://github.com/TarekRaafat/autoComplete.js/issues/206#issuecomment-826828343 and https://github.com/TarekRaafat/autoComplete.js/discussions/207 for some related inspiration and code.

But note that the code in those links is for older versions of autoComplete.js, so you can't just copy+paste, you need to understand. But there are some pretty images for inspiration.

**Reply by dhruv-icreative** [reply]
@folknor Thank you for the reply.

I got images to display in the result items, but when I select a particular item from the result list then the only name is displayed in the input field, not the image.

Do I need to append it to the value OR is there any other way to achieve it?
